docs/rules/no-shadow.md

```diff
@@ -50,12 +50,12 @@ This rule takes one option, an object, with properties `"builtinGlobals"`, `"hoi
 }
 ```
 
-### `builtinGlobals`
+### builtinGlobals
```

Comment:
Just want to confirm, are we now not going to use code style for these headings?